server/src/core/auth/dto/workspace-invite-hash.input.ts

```diff
@@ -0,0 +1,11 @@
+import { ArgsType, Field } from '@nestjs/graphql';
+import { IsNotEmpty, IsString, MinLength } from 'class-validator';
+
+@ArgsType()
+export class WorkspaceInviteHashValidInput {
+  @Field(() => String)
+  @IsString()
+  @IsNotEmpty()
+  @MinLength(10)
```

Comment:
Could we have shorter invite links that look user-friendly with a limited risk of collision